README.md

```diff
@@ -9,23 +13,29 @@
 <a href="#run-the-app-using-docker">using Docker</a>
 </p>
 
----
- 
-A GTFS Schedule (static) [General Transit Feed Specification (GTFS)](https://gtfs.mobilitydata.org/spec/gtfs-schedule) feed validator, maintained by [MobilityData](www.mobilitydata.org).
 
+<p align="center">
+<a href="https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md">☑️ list of rules implemented</a>
+</p>
 
-<video src="https://user-images.githubusercontent.com/63653518/182135478-f844cbfd-7037-4d02-bd65-f196cabaddeb.mp4" controls="controls" style="max-width: 730px;">
-</video>
+<p align="center">
+<a href="https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md"> 🤝 contribute to the project</a>
+</p>
+
+---
 
 This README contains information for the latest version of the project, which is under active development.  You can find the latest version of the validator application on the [Releases page](https://github.com/MobilityData/gtfs-validator/releases).
 
 # Introduction
 This is a cross-platform application written in Java that performs the following steps:
 1. Loads input GTFS zip file from a URL or disk.
-2. Checks file integrity, numeric type parsing and ranges as well as string format according to the [GTFS Schedule specification](https://gtfs.mobilitydata.org/spec/gtfs-schedule#h.hc443y62gb8c).
-3. Performs GTFS [business rule validation](/RULES.md).
+2. Checks file integrity, numeric type parsing and ranges.
+3. Performs complete validation against the [GTFS Schedule standard](https://gtfs.mobilitydata.org/spec/gtfs-schedule#h.hc443y62gb8c).
```

Comment:
Any reason not to use the gtfs.org link directly here?

Reply:
no reason, good catch!